mw-webapp/src/dataAccessLogic/DayReportDAL.ts

```diff
@@ -23,13 +23,11 @@ export class DayReportDAL {
   public static async getDayReports(wayUuid: string): Promise<DayReport[]> {
     const dayReportsUuids = (await WayService.getWayDTO(wayUuid)).dayReportUuids;
 
-    const dayReports = await Promise.all(dayReportsUuids.map(async (dayReportUuid) => {
-      const dayReport = await DayReportDAL.getDayReport(dayReportUuid);
+    const dayReports = await Promise.all(dayReportsUuids.map(DayReportDAL.getDayReport));
 
-      return dayReport;
-    }));
+    const dayReportSortedByDate = dayReports.sort((a, b) => b.date.getTime() - a.date.getTime());
 
-    return dayReports;
+    return dayReportSortedByDate;
```

Comment:
Do we really need to sort here? What if we want to add sorting by something else later? Maybe sort it where you get it

Reply:
We can't sort it in service layer right now because in our collections date is string but for sorting it we need to use timestamp. I added comment

Reply:
So you mean firebase can't sort strings?

Reply:
Yes firebase can sort by stroing but we need sort b date/ That why need a new issue about convert date-string to date-timestamp

Reply:
try again

Reply:
Changed methods on service layer for get all dayReports and used sort on firebase side by name